list.json

```diff
@@ -1071,9 +1071,14 @@
     "website": "https://liquidluck.readthedocs.io/en/latest/"
   },
   {
-    "name": "logya",
+    "created_at": "2011-08-21T00:00:00Z",
+    "description": "Logya is a static Web site generator written in Python designed to be easy to use and flexible.",
```

Comment:
Question: Not sure if `Web site` is TYPO or intentional ?
Suggestion: website

Otherwise this is LGTM 🎉